src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,50 @@
+package duke.task;
+
+/**
+ * A task that can be added to the list.
+ * Can be marked as done and has a description of the task.
+ */
+public class Task {
+    private Boolean done;
```

Comment:
```suggestion
    private Boolean isDone;
```
I saw that you use `isDone` in other places, is there a reason why `done` is used here instead?